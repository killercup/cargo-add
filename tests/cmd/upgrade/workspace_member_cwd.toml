bin.name = "cargo-upgrade"
args = ["upgrade", "libc"]
status = "success"
fs.sandbox = true
fs.cwd = "workspace_member_cwd.in/one"

[env.add]
CARGO_IS_TEST="1"
