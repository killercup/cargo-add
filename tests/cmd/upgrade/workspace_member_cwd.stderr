    Upgrading libc v0.2.28 -> v99999.0.0
